### ISSUE TITLE
`nsamples` in the `tx_indiv` should accept a single number, or index of patients.

### ISSUE THREAD

**Issue by SimonGoring** [johnwilsonICON/txvis#22]
There may be reasons to want to view a fixed subset of patients.  It would be nice to be able to pass:

`tx_indiv(object, nsamples=c("PT1", "PT2", "PT3"))`

or some such similar example.

**Comment by sgoring** [comment]
Yes, or they can just subset their dataset and create a new txvis object.

**Comment by SimonGoring** [comment]
Then we need to create a `subset` method for `txvis`. Which do we prefer?  I think subset might be useful.  By patient code (or part of a patient code?), by treatment (all patients with treatment 2), by start or end date. . .

**Comment by sgoring** [comment]
I was thinking more along the lines that the person could create a new txvis object using a subset of their data. Meaning that we don't have to do anything. :-) Maybe a v2.0 feature?
